### DISCUSSION TITLE
Shouldn't publicly stated security changes prompt a rather quick release?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3462] [Q&A Feedback] [answered]
The changelog lists a FIXED SECURITY ISSUE ID, but there was no release of any kind shortly after.

Shouldn't there be a release, shortly after something that got a SECURITY ISSUE ID got fixed?

Is that purely due to the low severity (which I just assume to be low)?

**Comment by DavidXanatos** [comment, marked as answer]
As noted in the release notes for 1.12.2 there is a signed release 1.12.3 scheduled for the end fo the week, i want first to ensure all the changes in 1,12,2 do not cause any issues.

**Reply by bastik-1001** [reply]
I understand.
